tests/filesystem_tests/test_hdfs_handler.py

```diff
@@ -208,10 +210,42 @@ def test_remove(self):
             self.assertFalse(handler.exists(nested_dir))
             self.assertFalse(handler.exists(nested_file))
 
-    def test_stat(self):
-        # pass for now
-        # TODO(tianqi) add test after we well defined the stat
-        pass
+    def test_stat_file(self):
+        timestamp_eps = 2.0
+        test_file_name = "testfile"
+        with pfio.create_handler(self.fs) as handler:
+            ts = datetime.datetime.now().timestamp()
+            with handler.open(test_file_name, 'w') as fp:
+                fp.write('foobar')
+
+            stat = handler.stat(test_file_name)
+            self.assertIsInstance(stat, HdfsFileStat)
+            self.assertTrue(stat.filename.endswith(test_file_name))
+            self.assertEqual(stat.size, 6)
+            self.assertFalse(stat.isdir())
+            self.assertEqual(stat.mode, 0o100644)
+            self.assertTrue(abs(stat.last_accessed - ts) < timestamp_eps)
+            self.assertTrue(abs(stat.last_modified - ts) < timestamp_eps)
+
```

Comment:
Can you also test the rest of `HdfsFileStat`, you can use the data in the from the `info()`